### PR TITLE
Configured API endpoint to create political party

### PR DESCRIPTION
**Pull request by jaystaks**
### What does this PR do?
Allows the admin to create a party

### Description of Task to be completed?

   1.  A party is to be created from the v1/party POST api endpoint

party_post

### CODE REVIEW

File: politicoAPI/app/api/v1/views/politicalparty.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+class Party():
+    @politicalparty.route('/parties',methods=['POST'])
+    def create_political_party():
+        party = request.get_json()
+        name = party['name']
+        hqAddress = party['hqAddress']
+        logoUrl = party['logoUrl']
+
+        feedback = Political().create_political_party('id', 'name', 'hqAddress', 'logoUrl')
```
Comment (Adoniswalker) [inline politicoAPI/app/api/v1/views/politicalparty.py:15]: What data are you passing through? is it variables or hardcoded ones

File: politicoAPI/app/api/v1/models/political_model.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+parties = []
+
+class Political:
+    def __init__(self):
+        self.data= parties
+
+    def create_political_party(self, id, name, hqAddress, logoUrl):
+        party= {
+            "party_id": len(self.data)+1,
```
Comment (Adoniswalker) [inline politicoAPI/app/api/v1/models/political_model.py:9]: You have not done any validations

File: politicoAPI/app/api/v1/views/politicalparty.py (hunk 1 above)
Comment (jaystaks) [reply politicoAPI/app/api/v1/views/politicalparty.py:15]: i rectified that part

File: politicoAPI/app/api/v1/models/political_model.py (hunk 2 above)
Comment (jaystaks) [reply politicoAPI/app/api/v1/models/political_model.py:9]: im working on them

**Pull request merged**